netutils/config/parser.py

```diff
@@ -12,34 +13,34 @@
 class BaseConfigParser:  # pylint: disable=too-few-public-methods
     """Base class for parsers."""
 
-    comment_chars = ["!"]
-    banner_start = ["banner", "vacant-message"]
+    comment_chars: t.List[str] = ["!"]
+    banner_start: t.List[str] = ["banner", "vacant-message"]
```

Comment:
These type hints are not needed, Mypy is able to infer this type correctly. I think we should drop them, as then they just add noise to the code.